CordovaLib/Classes/Private/Plugins/CDVIntentAndNavigationFilter/CDVIntentAndNavigationFilter.h

```diff
@@ -20,11 +20,7 @@
 #import <Cordova/CDVPlugin.h>
 #import <Cordova/CDVWhitelist.h>
 
-#if WK_WEB_VIEW_ONLY
 #define CDVWebViewNavigationType int
```

Comment:
Does this get used anywhere by WKWebView or should it be removed entirely?

Reply:
`CDVWebViewNavigationType` is used here.

https://github.com/erisu/cordova-ios/blob/refactor/swap-webviews/CordovaLib/Classes/Private/Plugins/CDVIntentAndNavigationFilter/CDVIntentAndNavigationFilter.m#L105-L132

https://github.com/erisu/cordova-ios/blob/refactor/swap-webviews/CordovaLib/Classes/Private/Plugins/CDVIntentAndNavigationFilter/CDVIntentAndNavigationFilter.m#L134-L137